### ISSUE TITLE
how to apply my own wind data to inflow wind module?

### ISSUE THREAD

**Issue by 1Naruto1** [OpenFAST/openfast#1532]
Dear Developer,

I would like to ask the question about "how to apply my own wind data to inflow wind module?" For example, I have a series of wind speed and direction data over time, and I want to apply it to inflow wind module. I tried to set the wind type to 2, and defined a file ended with .dat, which follows the file "inflowwind_uniform_example.dat" that I downloaded from your website. Then I changed the parameters setting (Filename_Uni) to the file name I defined before in the section of inflow wind module "Parameters for Uniform wind file   [used only for WindType = 2]". However, it does NOT work, because in the output file, the wind speed is same as nothing to change.

Kind Regards

**Comment by jjonkman** [comment]
Dear @1Naruto1,

It sounds like you are following the correct process.  Did you save all of the files you changed before executing the OpenFAST simulation?  Have you enabled `Echo` in InflowWind to identify issues in the input file processing?

Best regards,

**Comment by 1Naruto1** [comment]
Dear @jjonkman,

Thanks for your reply! I do save all the files that I changed before running OpenFAST. Then I followed your suggestion to enable Echo.
![image](https://user-images.githubusercontent.com/106254703/233251066-09eda443-6455-4f3f-9478-961a0f674e99.png)

I found the inflowwind input file is turbulent winds, although I set the wind type to 2. I am wondering that if I need set the inflow file to "inflowwind_uniform_example.dat" that I defined by my own data in the file that end with .fst directly?
![image](https://user-images.githubusercontent.com/106254703/233251627-d9b2c7cf-5602-449e-8fdb-e1e9715feb22.png)

Looking forward to your comments :)

Kind Regards,
